UN-OCHA/common-design-site
# CD-505 Update e2e tests

This incorporates changes to the base theme and the sub theme https://github.com/UN-OCHA/common_design/pull/431

### To test:
1. Check out the branch and go to `themes/custom/common_design_subtheme/`
2. Follow the instructions in the README to run the tests.

This does not deal with the lack of tests running as part of the CI. We had the tests running in Travis but no longer. A separate issue can be opened to integrate these tests with Github Actions.
